File: src/main/java/marxls/Mapping.java
package marxls;

import java.util.List;

import lombok.Data;

@Data
public class Mapping {
  private String name;
  private String sheet;
  private String className;
  private List<Member> members;

  public Member getMember(String titleOrColumn) {
    return members.stream()
        .filter(member -> member.isTitleBased() ? titleOrColumn.equals(member.getTitle())
            : titleOrColumn.equals(member.getColumn()))
        .findFirst().orElse(null);
  }
}

File: src/main/kotlin/net/technearts/Mapping.kt
package net.technearts

class Mapping {
    var name: String? = null
    var sheet: String? = null
    var className: String? = null
    var members: List<Member>? = null

    fun getMember(titleOrColumn: String): Member? {
        return members?.stream()?.filter { member ->
            if (member.isTitleBased)
                titleOrColumn == member.title
            else
                titleOrColumn == member.column
        }?.findFirst()?.orElse(null)
    }
}
